src/napari/utils/logo.py

```diff
@@ -0,0 +1,56 @@
+import re
+from datetime import date
+from importlib import resources
+from pathlib import Path
+
+
+def available_logos():
+    logo_dir = Path(resources.files('napari').joinpath('resources', 'logos'))
+    variants = ['auto']
+    for logo in Path(logo_dir).glob('*.svg'):
+        if match := re.match(r'(.*)-plain-light', logo.stem):
+            variants.append(match.group(1))
+    return variants
```

Comment:
```suggestion
    return sorted(variants) 
```
`glob` order is not deterministic or sort glob output before iterating to ensure that `"auto"` is first.